pubspec.lock

```diff
@@ -63,7 +63,7 @@ packages:
       name: async
```

Comment:
remove this file also from commit.

Reply:
okay @hiashutoshsingh

Reply:
@hiashutoshsingh  Can you guide me how to remove the file from commit ??

Reply:
make a copy of the file at another place and delete that file then commit it.